packages/analytics-js/src/components/pluginsManager/defaultPluginsList.ts

```diff
@@ -5,14 +5,14 @@ import { PluginName } from './types';
  */
 const defaultOptionalPluginsList: PluginName[] = [
   PluginName.ConsentManager,
-  PluginName.DataplaneEventsQueue,
   PluginName.DeviceModeDestinations,
   PluginName.ErrorReporting,
   PluginName.ExternalAnonymousId,
   PluginName.GoogleLinker,
   PluginName.NativeDestinationQueue,
   PluginName.StorageEncryptionLegacy,
   PluginName.XhrQueue,
+  PluginName.BeaconQueue,
```

Comment:
this should not be in the defaults, we should add it in only if beacon is selected (and remove XhrQueue in this case).

this is done in plugin manager at getPluginsToLoadBasedOnConfig method